commands/user

```diff
@@ -120,20 +122,49 @@ function cmd_list {
 
 # @sudo cmd_delete admin
 # @doc cmd_delete
-# Delete user
-#   USERNAME    User account name
+# Delete users
+#   -y, --yes       Do not ask for confirmation
+#   USERNAME...     User account name (multiple names can be added).
 function cmd_delete {
-  if [[ $# -eq 0 ]] || [[ -z "$1" ]]; then
+  local yes=""
+  local names=()
+
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      -y | --yes) yes="y";;
```

Comment:
What if there is a user with name '--yes'? (and his brother 'DROP TABLE' =)

Reply:
These are not allowed user names. It's denied to create with our CLI and I think we shouldn't handle them.

Reply:
I would say that it will mean you wrote a bad regexp in 0eec5fe0354e22705b98a1f5bf933b228ca3bebe. ;)
But it seems to me that regexp looks fine.